Create includes for HTTP event postback and new user guide article (#8867)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/transactional_message_api_campaign/tracking_transactional_emails.md

---
nav_title: Tracking Transactional Emails
article_title: Tracking Transactional Emails
page_order: 1
description: "This reference article covers how to set up real-time tracking for transactional email campaigns."
page_type: reference
tool:
  - Campaigns
channel: email

---

# Tracking transactional emails

> This page describes how to set up real-time tracking for [transactional email campaigns][1]. For more information about the endpoint itself, refer to [Send transactional emails using API-triggered delivery][2].

> When you send transactional emails—like order confirmations or password resets—it’s essential to know whether they reach your customers. With Braze’s transactional HTTP event postbacks, you’ll get real-time insights into the status of every transactional email, so you can act quickly if there’s an issue.

Author: ```suggestion
When you send transactional emails—like order confirmations or password resets—it’s essential to know whether they reach your customers. With Braze transactional HTTP event postbacks, you’ll get real-time insights into the status of every transactional email, so you can act quickly if there’s an issue.
```

Use this feature to:

- **Monitor your emails in real-time:** Immediately see if messages are sent, processed, delivered, or encounter issues.

> - **Respond proactively:** Retry messages, switch to another channel like SMS, or use fallback systems to ensure your communications are delivered.

Author: ```suggestion
- **Respond proactively:** Retry messages, switch to another channel like SMS, or use fallback systems to make sure your communications are delivered.
```

> ## How it works

Reviewer: ```suggestion
## Tracking your transactional emails
```

thoughts?
